pkgs/data/fonts/font-awesome-4/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchzip }:
+let
+  version = "4.7.0";
+in fetchzip rec {
+  name = "font-awesome-${version}";
```

Comment:
Can you add a concrete example, why we have this package? Then we can reevaluate in future if we still need the old version.